### ISSUE TITLE
nix develop: hash mismatch in fixed-output derivation

### ISSUE THREAD

**Issue by jademackay** [JuliaCN/Julia2Nix.jl#32]
I am a Nix novice so the the following might not be an issue, however, in the `Julia2Nix.jl` current latest version of main (commit 4983f9101de99d37e562f28b508dba4bd89b8829) `nix develop` fails with 
```
error: hash mismatch in fixed-output derivation '/nix/store/if8hv0v7204xa19jhn9syng8ir4xnznf-julia-latest-linux-x86_64.tar.gz.drv':
         specified: sha256-RasDQSrcq62R4HM+RDP2QuFeHbpemWtfnZLU/Y8VR+M=
            got:    sha256-bC7y+zep1guINMGfEwDIHZ1dU9vLQiGp+5IoQuF5oow=
```
I overcame the problem by issuing
```
nix develop .#update --command nvfetcher-update
```
but I suspect this is not the desired practice? ... Perhaps an out of date CICD setting?

**Comment by GTrunSec** [comment]
really? did you try `nix flake nix flake lock --update-input julia2nix`.  this issue has been fixed by previous commits.

**Comment by GTrunSec** [comment]
Also, the CI can not reproduce this issue as well.  there is no cache in our CI Env.

**Comment by GTrunSec** [comment]
thanks, fixed by #33

**Comment by jademackay** [comment]
Yep, nvfetcher 93aacd14b075a08c68c439baedfe69dcf6c01ac5 resolves the issue for me too. Thanks!
